lib/plaid/models.rb

```diff
@@ -578,6 +578,90 @@ class Institution < BaseModel
       # Public: The Array of String product names supported by this institution.
       # E.g. ["auth", "balance", "identity", "transactions"].
       property :products
+
+      ##
+      # :attr_reader:
+      # Public: The Brand name of this institution.
+      # E.g. "First Platypus Bank"
+      property :brand_name
+
+      ##
+      # :attr_reader:
+      # Public: The brand subheading of this institution.
+      property :brand_subheading
+
+      ##
+      # :attr_reader:
+      # Public: A string containing the name break of this institution.
+      property :name_break
+
+      ##
+      # :attr_reader:
+      # Public: ???
+      property :portal
+
+      ##
+      # :attr_reader:
+      # Public: A hash of colors associated with this institution.
+      # E.g. {"dark"=>"#006692", "darker"=>"#00456f", "light"=>"#378fbe",
+      #       "primary"=>"#0075a3"}
+      property :colors
```

Comment:
@joshuamcginnis since there are other properties that are coerced into objects, it's likely the authors will want that here. An example to follow would be the `data` property on the `IdentityAddress` model.

Reply:
Near - I wasn't aware of the `coerce` option in models. This has been updated.

Reply:
Nicely done, thanks! 👍🏻